torch/CMakeLists.txt

```diff
@@ -70,6 +70,7 @@ set(TORCH_PYTHON_INCLUDE_DIRECTORIES
 
     ${TORCH_ROOT}/third_party/gloo
     ${TORCH_ROOT}/third_party/onnx
+    ${TORCH_ROOT}/third_party/flatbuffers/include
```

Comment:
LGTM, but in followup PR can we limit it to just 2 files in question? (So that we don't leak flatbuffers dependency to module extensions)

Reply:
Do you mean we can use different include paths when compiling `flatbuffer_loader.cpp` and `flatbuffer_serializer.cpp` but not when compiling other .cpp files? How do I do that? Also it's not just those 2 files also ones that happens to include `flatbuffer_loader.h` or `flatbuffer_serializer.h`.